datafusion/core/src/physical_plan/aggregates/row_hash.rs

```diff
@@ -75,19 +75,10 @@ use hashbrown::raw::RawTable;
 /// [Compact]: datafusion_row::layout::RowType::Compact
 /// [WordAligned]: datafusion_row::layout::RowType::WordAligned
 pub(crate) struct GroupedHashAggregateStream {
-    stream: BoxStream<'static, ArrowResult<RecordBatch>>,
-    schema: SchemaRef,
-}
-
-/// Actual implementation of [`GroupedHashAggregateStream`].
-///
-/// This is wrapped into yet another struct because we need to interact with the async memory management subsystem
```

Comment:
this comment about another struct for memory management is out of date and so I folded `GroupedHashAggregateStreamInner` directly into `GroupedHashAggregateStream`